addons/armaos/functions/fnc_os_date.sqf

```diff
@@ -1,38 +1,31 @@
-params ["_options", "_consoleInput"];
+/**
+ * Prints/outputs the ingame date and time on a given computer.
+ * Also returns informations about the success of the command.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ * 2: Options <[STRING]>
+ *
+ * Results:
+ * 1: Informations/Date and Time <[STRING]>
+ */
 
-_result = [];
+params ["_computer", "_options"];
 
-_optionsCount = count _options;
+if (count _options >= 1) exitWith {["   Command: date has no options"];};
 
-scopeName "main";
+_date = date;
```

Comment:
private

Reply:
changed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763